code/modules/antagonists/cult/blood_magic.dm

```diff
@@ -182,31 +182,32 @@
 	health_cost = 12
 
 /datum/action/innate/cult/blood_spell/equipment
-	name = "Summon Equipment"
-	desc = "Allows you to summon a ritual dagger, or empowers your hand to summon combat gear onto a cultist you touch, including cult armor, a cult bola, and a cult sword."
+	name = "Summon Combat Equipment"
+	desc = "Empowers your hand to summon combat gear onto a cultist you touch, including cult armor, a cult bola, and a cult sword. Not recommended for use before the blood cult's presence has been revealed."
 	button_icon_state = "equip"
 	magic_path = "/obj/item/melee/blood_magic/armor"
 
-/datum/action/innate/cult/blood_spell/equipment/Activate()
-	var/choice = alert(owner,"Choose your equipment type",,"Combat Equipment","Ritual Dagger","Cancel")
-	if(choice == "Ritual Dagger")
-		var/turf/T = get_turf(owner)
-		owner.visible_message("<span class='warning'>[owner]'s hand glows red for a moment.</span>", \
-			"<span class='cultitalic'>Red light begins to shimmer and take form within your hand!</span>")
-		var/obj/O = new /obj/item/melee/cultblade/dagger(T)
-		if(owner.put_in_hands(O))
-			to_chat(owner, "<span class='warning'>A ritual dagger appears in your hand!</span>")
-		else
-			owner.visible_message("<span class='warning'>A ritual dagger appears at [owner]'s feet!</span>", \
-				 "<span class='cultitalic'>A ritual dagger materializes at your feet.</span>")
-		SEND_SOUND(owner, sound('sound/effects/magic.ogg',0,1,25))
-		charges--
-		desc = base_desc
-		desc += "<br><b><u>Has [charges] use\s remaining</u></b>."
-		if(charges<=0)
-			qdel(src)
-	else if(choice == "Combat Equipment")
-		..()
+/datum/action/innate/cult/blood_spell/dagger
+	name = "Summon Ritual Dagger"
+	desc = "Allows you to summon a ritual dagger, in case you've lost the dagger that was given to you."
+	invocation = "Wur d'dai leev'mai k'sagan!" //where did I leave my keys, again?
+	button_icon_state = "equip" //this is the same icon that summon equipment uses, but eh, I'm not a spriter
+
+/datum/action/innate/cult/blood_spell/dagger/Activate()
+	var/turf/T = get_turf(owner)
+	owner.whisper(invocation, language = /datum/language/common)
+	owner.visible_message("<span class='warning'>[owner]'s hand glows red for a moment.</span>", \
+		"<span class='cultitalic'>Your plea for aid is answered, and light begins to shimmer and take form within your hand!</span>")
+	var/obj/O = new /obj/item/melee/cultblade/dagger(T)
+	if(owner.put_in_hands(O))
+		to_chat(owner, "<span class='warning'>A ritual dagger appears in your hand!</span>")
+	else
+		owner.visible_message("<span class='warning'>A ritual dagger appears at [owner]'s feet!</span>", \
+			 "<span class='cultitalic'>A ritual dagger materializes at your feet.</span>")
+	SEND_SOUND(owner, sound('sound/effects/magic.ogg',0,1,25))
+	charges--
+	if(charges<=0)
```

Comment:
```suggestion
	var/turf/onwer_turf = get_turf(owner)
	owner.whisper(invocation, language = /datum/language/common)
	owner.visible_message("<span class='warning'>[owner]'s hand glows red for a moment.</span>", \
		"<span class='cultitalic'>Your plea for aid is answered, and light begins to shimmer and take form within your hand!</span>")
	var/obj/item/melee/cultblade/dagger/summoned_blade = new (onwer_turf)
	if(owner.put_in_hands(summoned_blade))
		to_chat(owner, "<span class='warning'>A ritual dagger appears in your hand!</span>")
	else
		owner.visible_message("<span class='warning'>A ritual dagger appears at [owner]'s feet!</span>", \
			 "<span class='cultitalic'>A ritual dagger materializes at your feet.</span>")
	SEND_SOUND(owner, sound('sound/effects/magic.ogg', FALSE, 0, 25))
	charges--
	if(charges <= 0)
```

Reply:
"onwer_turf"

Reply:
also, why'd you change that "0,1" to a "FALSE, 0"? Shouldn't it be a "FALSE, 1"?

Reply:
Use better variable names than single upper-case letters.
As for the zero, it's in the reference:
```
Format:
	sound(file, repeat = 0, wait, channel, volume)
	(supports named arguments)
Args:
	file: A sound file to play
	repeat: 1 to play sound repeatedly
	wait: 0 to interrupt current sound on channel; 1 to wait in queue
	channel: 0 for any available channel, 1-8 for specific channel (wavs only)
	volume: 100 for full volume (default), 0 for none, or any value in between
```
This is an `.ogg` not a `.wav`

Reply:
okay, I'll trust you on this one